md/building-community-edition-from-source.md

```diff
@@ -29,6 +29,6 @@ Note that you can run Bonita BPM on Windows but you cannot build Bonita BPM on W
 
 In order to build Bonita BPM you need to use the script provided in [Build Bonita BPM](https://github.com/Bonitasoft-Community/Build-Bonita-BPM) GitHub project. Use the branch of the project that match the version you want to build.
 
-Simply run `bash build-script-7.3.2.sh` and provide required information.
+Simply run `bash build-script-7.4.1.sh` and provide required information.
```

Comment:
We need to do the update manually because connectors version to build and includes in a version cannot be automated.
So until improvement on this I prefer to manually update documentation page each time we update the build script.